### DISCUSSION TITLE
mixamo 커스텀 애니메이션 적용

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#538] [5. Q&A]
mixamo 커스텀 애니메이션 적용이 가능한가요?

https://www.mixamo.com/#/
![image](https://user-images.githubusercontent.com/50188795/160513965-10003756-6323-4bb0-b492-fa7e2c0a8ee5.png)

**Comment by x4cf** [comment]
인스팩터에서 휴머노이드설정후 사용가능합니다 :D
저도 프로젝트에 믹사모 애니메이션을 사용하고 있어요

*믹사모 라이센스에 유의해주세요

<img width="950" alt="147799753-7895b937-ad0f-4459-b581-f3e7590414ad" src="https://user-images.githubusercontent.com/90679926/160514322-6d184ef7-ba67-48a8-9761-9b64cdee7e4a.png">
<img width="950" alt="147799756-58da677f-8d48-468a-8f7c-3ac34acb9c60" src="https://user-images.githubusercontent.com/90679926/160514329-7e041f2b-a9ef-4231-ac1b-d3078b3d99da.png">

**Reply by smilejsu82** [reply]
믹사모 라이센스란 구체적으로 어떤것인가요?

**Reply by x4cf** [reply]
믹사모 리소스를 사용했다면 콘탠츠에 믹사모의 리소스를 사용했다는걸 명시해야합니다

**Reply by smilejsu82** [reply]
![image](https://user-images.githubusercontent.com/50188795/160514670-cba1700d-fcb7-4ba4-93d8-4a25a801bbb9.png)
![image](https://user-images.githubusercontent.com/50188795/160514724-e6b73392-26a0-4870-b290-fe4fcebeaa99.png)


아하 이런 내용이군요  답변 감사합니다

**Reply by x4cf** [reply]
적절한 자료첨부좋아요 XD
이 질문을 보는 다음사람에게도 도움이 될 수 있을것같아요 :)

**Reply by smilejsu82** [reply]
정말 매번 놀랍니다 빠른 답변 !!!

**Reply by smilejsu82** [reply]
https://user-images.githubusercontent.com/50188795/160521219-fdc4dd23-fa5a-4ec3-aa20-eaa031bf4b94.mp4

잘동작 합니다 감사합니다
